website/pages/docs/k8s/helm.mdx

```diff
@@ -730,9 +765,118 @@ and consider if they're appropriate for your deployment.
         }
       ```
 
+- `meshGateway` ((#v-meshgateway)) - Configure mesh gateways.
+
+  - `enabled` ((#v-meshgateway-enabled)) (`boolean: true`) - If mesh gateways are enabled, a Deployment will be created that runs
+    gateways and Consul Connect will be configured to use gateways.
+    See https://www.consul.io/docs/connect/mesh_gateway.html
+    Requirements: Consul 1.6.0+ and consul-k8s 0.9.0+ if using
```

Comment:
doesn't it require the latest consul-k8s for the `service-address` command?

Reply:
🍺